### ISSUE TITLE
PVE DSM VM hang on while "Booting the kernel"

### ISSUE THREAD

**Issue by xiaxinzhe** [fbelavenuto/arpl#359]
![image](https://user-images.githubusercontent.com/61304326/204344577-04f854e1-d47b-47b2-8733-389aec231862.png)
![image](https://user-images.githubusercontent.com/61304326/204344677-14a101e2-1fee-48ac-9d19-d078bd8badc8.png)
I chose the 918+ and the latest version of DSM.
My physical machine is as follow:
![image](https://user-images.githubusercontent.com/61304326/204345105-42810843-bccc-48a2-abac-b22222aa1fa4.png)
What caused this problem? I can't find any detail info or help on search engine, help me please : (

**Comment by AuxXxilium** [comment]
Same for me in 3 different setup. It looks like the issue is related to this: f6dbd4b8cd88ddf06efe7e10302d51968b5d703d

**Comment by OrpheeGT** [comment]
You should not have any disk in ide (ide0) all disks even the loader should be sata disks.
It may not fix the issue, but at least it would be better configured.

**Comment by fbelavenuto** [comment]
Please configure the loader image as SATA disk, not IDE

**Comment by AuxXxilium** [comment]
SATA isn't working, too.

**Comment by MartinKuhl** [comment]
The loader in my case is configured as SCSI but this was working for month and was working for all previous versions of ARPL but not with the newest one.

**Comment by AuxXxilium** [comment]
Yes, it wont boot through SATA. Same on baremetal install. Latest version boot is broken. The problem is related to the commit i posted above. Without this it is working.

**Comment by fbelavenuto** [comment]
This commit is only for the new docker image creation.
I've recompiled all modules with official synology toolkit and this maybe a cause of troubles

**Comment by AuxXxilium** [comment]
Is it possible that the problem is inside. I compiled without this commit and it is working.

**Comment by MartinKuhl** [comment]
For me in beta5 nothing changed and I am still not able to boot DSM.

**Comment by fbelavenuto** [comment]
Please do a test, add the addon dbgutils, rebuild and test it again

**Comment by MartinKuhl** [comment]
Hi @fbelavenuto yes with the addon dbgutils not my VM is boot normal again!

**Comment by fbelavenuto** [comment]
Ok, some trouble with libraries!! I'll investigate more, thanks for the test (you can continue using dbgutils with no trouble)

**Comment by fbelavenuto** [comment]
Guys, please update all (ARPL, addons and modules), rebuild and test it.

**Comment by C2BB** [comment]
Still stuck at "Booting the Kernel" with Beta-6. Beta-3 is the last working version with my bare-metal install.

**Comment by MartinKuhl** [comment]
Beta 6 is working for me even without the addon dbgutils

**Comment by MartinKuhl** [comment]
Do anyone else have these error message?

**Comment by fbelavenuto** [comment]
@xiaxinzhe Still problem?

**Comment by fbelavenuto** [comment]
Please update ARPL, reboot, update modules, addons and lkm, rebuild the loader and test it.
